srt-protocol/src/protocol/receiver/buffer.rs

```diff
@@ -158,6 +164,19 @@ impl MessagePacketCount {
     }
 }
 
+#[derive(Debug, Default)]
+pub struct ReceiveBufferStatistics {
```

Comment:
I don't see this used anywhere??

Reply:
This is dead code, left over from an early iteration on the design, I'll delete.